packages/cli-hydrogen/src/cli/commands/hydrogen/preview.ts

```diff
@@ -16,6 +16,12 @@ export default class Preview extends Command {
       default: '3000',
       env: 'SHOPIFY_FLAG_PORT',
     }),
+    env: Flags.string({
+      char: 'e',
+      description: 'the file path to your .env',
+      default: '',
```

Comment:
I'd also recommend normalizing the path like we are doing [here](https://github.com/Shopify/cli/blob/main/packages/app/src/cli/flags.ts#L12). Otherwise you'll be passing down a string that can be either a relative or an absolute path and you'll be forcing deeper components to make assumptions about how to make the relative path absolute.